test/active_record_doctor/detectors/missing_presence_validation_test.rb

```diff
@@ -91,6 +91,16 @@ def test_non_null_boolean_is_not_reported_if_nil_excluded
     refute_problems
   end
 
+  def test_non_null_boolean_is_reported_if_exclusion_is_proc
```

Comment:
@fatkodima, please fix the test case name (I think it should say `..._is_not_reported_...`) and it's good to merge.

Reply:
@gregnavis Oops. Fixed.